python_scripts/publications.py

```diff
@@ -120,6 +122,8 @@ def write_publication(data):
             person_card = getStaff(apiBase, staff, author_id)
             if person_card != None:
                 author_info.append(person_card)
+        else:
+            author_info.append(defaultdict(lambda: "Outsider"))
```

Comment:
Shall we 'append' the following other than the string, "Outsider"?

```json
 {
        "type": "OUTSIDER",
        "id": "",
        "name": "",
        "email": "",
        "profile_image": "#",
        "profile_url": "#"
 }
```

Reply:
Anything is fine. We just need a placeholder for that location.